### PR TITLE
fix(medusa-js): Use correct Payload type in resetPassword

### PR DESCRIPTION
**Pull request by sabakhilji**
Method resetPassword in the Customers domain of medusa-js changed to StorePostCustomersResetPasswordReq

### CODE REVIEW

File: packages/medusa-js/src/resources/customers.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ class CustomerResource extends BaseResource {
    * @return {ResponsePromise<StoreCustomersRes>}
    */
   resetPassword(
-    payload: StorePostCustomersCustomerPasswordTokenReq,
+    payload: StorePostCustomersResetPasswordReq,
```
Comment (olivermrbl) [inline packages/medusa-js/src/resources/customers.ts:86]: todo: Add import similar to how other types are imported

**Review by olivermrbl — CHANGES_REQUESTED** [review]
Appreciate the contribution, just have a single comment 💪

File: packages/medusa-js/src/resources/customers.ts (hunk 1 above)
Comment (sabakhilji) [reply packages/medusa-js/src/resources/customers.ts:88]: done:)

**Comment by sabakhilji** [comment]
> Can I get you to open a new PR with `develop` as base branch?

Do I need to open a new PR ?

**Comment by sabakhilji** [comment]
> Can I get you to open a new PR with `develop` as base branch?

Hi @olivermrbl do I need to open a new PR with `develop` as the base branch or can I update this branch?

**Comment by olivermrbl** [comment]
Duplicate of #2044

**Pull request closed without merging**